gp-includes/route.php

```diff
@@ -168,8 +168,8 @@ public function invalid_nonce_and_redirect( $action, $url = null ) {
 	 *                                 Default: 'You are not allowed to do that!'.
 	 * @return false
 	 */
-	public function can_or_forbidden( $action, $object_type = null, $object_id = null, $message = 'You are not allowed to do that!' ) {
-		$can = $this->can( $action, $object_type, $object_id );
+	public function can_or_forbidden( $action, $object_type = null, $object_id = null, $extra = null, $message = 'You are not allowed to do that!' ) {
```

Comment:
The `$extra` should be at the end of the list, otherwise it's an API breaking change for any plugins that use the code.